content/en/docs/Kyverno CLI/_index.md

```diff
@@ -210,6 +210,107 @@ policies:
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
 
+Value files also support global values, which can be passed to all resources the policy is being applied to.
+
+Format of `value.yaml`:
+
+```yaml
+policies:
+  - name: <policy1 name>
+    resources:
+      - name: <resource1 name>
+      values:
+        <variable1 in policy1>: <value>
+        <variable2 in policy1>: <value>
+      - name: <resource2 name>
+      values:
+        <variable1 in policy1>: <value>
+        <variable2 in policy1>: <value>
+  - name: <policy2 name>
+    resources:
+      - name: <resource1 name>
+      values:
+        <variable1 in policy2>: <value>
+        <variable2 in policy2>: <value>
+      - name: <resource2 name>
+      values:
+        <variable1 in policy2>: <value>
+        <variable2 in policy2>: <value>
+globalValues:
+  <global variable1>: <value>
+  <global variable2>: <value>
+```
+
+If a resource-specific value and a global value have the same variable name, the resource value takes precedence over the global value. See the pod `test-global-prod` in the following example.
+
+Example:
+
+Policy manifest (`add_dev_pod.yaml`):
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: cm-globalval-example
+spec:
+  validationFailureAction: enforce
+  background: false
+  rules:
+    - name: validate-mode
+      match:
+        resources:
+          kinds:
+            - Pod
+      validate:
+        message: "The value {{ request.mode }} for val1 is not equal to 'dev'."
+        deny:
+          conditions:
+            - key: "{{ request.mode }}"
+            operator: NotEquals
```

Comment:
The indentation for `operator` and `value` are not aligned with `key`.

Reply:
Hi @chipzoller, thanks for the review. Just submitted the changes.